frontend/src/styles/Catalog.scss

```diff
@@ -0,0 +1,12 @@
+.catalog {
+  margin-top: 20px;
+
+  > .row {
+    margin: 20px;
+  }
+
+  > .pagination {
```

Comment:
Resource for the curious: https://css-tricks.com/the-sass-ampersand/

Reply:
very cool resource :scroll: